packages/icons/src/react/SvgIcon.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import icons from '../constants/icons';
+
+interface IconProps {
+  name: string;
+  size: number;
+}
+
+export const SvgIcon = ({ name, size, ...restProps }: IconProps): JSX.Element => {
+  return (
+    <svg viewBox="0 0 24 24" fill="currentColor" width={size || '24'} height={size || '24'} {...restProps}>
+      <path d={icons[name]} fill="currentColor" />
```

Comment:
and here should print all the children from the constants/SVG. CurrentColor is already in there.

Reply:
This is deprecated. I have chosen a better way to work with. See the follow-up PR.